tc_driver/Cases.jl

```diff
@@ -312,19 +290,18 @@ function initialize_profiles(
     @. p_c = prof_p(z)
 end
 
-function surface_params(case::Bomex, surf_ref_thermo_state, thermo_params)
-    p_f_surf = TD.air_pressure(thermo_params, surf_ref_thermo_state)
-    ρ_f_surf = TD.air_density(thermo_params, surf_ref_thermo_state)
-    FT = eltype(p_f_surf)
+function surface_params(case::Bomex, thermo_params)
+    FT = eltype(thermo_params)
     zrough::FT = 1.0e-4
+    psurface::FT = 1.015e5
     qsurface::FT = 22.45e-3 # kg/kg
-    θ_surface::FT = 299.1
+    θ_surface::FT = 299.1 # PAPER CHECK: Should we use Tsurface = 300.4 instead?
```

Comment:
Both are mentioned in the Bomex paper: 

"For surface boundary conditions an observed surface pressure of 1015 hPa is prescribed. The observed sea surface temperature was 300.4 K, implying a sea surface potential temperature of 299.1 K and a sea surface saturation specific humidity of 22.45 g/kg."

And I checked that they are indeed equivalent. The case is flagged in the CI as slightly different but the differences are very small (aka acceptable) for all the Bomex variants we are running.